src/components/fpl/ControlsBar.tsx

```
@@ -0,0 +1,153 @@
+        </button>
+
+        {/* Start/Stop Live Button */}
+        <button
+          onClick={isPolling ? onStopPolling : onStartPolling}
+          disabled={loading}
+          className={`flex items-center gap-1 text-sm font-medium py-1.5 px-3 rounded transition-colors duration-200 ${
+            isPolling
+              ? "bg-red-600 hover:bg-red-700 disabled:bg-red-300"
+              : "bg-blue-600 hover:bg-blue-700 disabled:bg-blue-300"
+          } text-white`}
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)